lib/src/utils/crypto/base64.dart

```diff
@@ -0,0 +1,19 @@
+
+extension Base64Utils on String {
+  String toBase64() {
+    return replaceAll('-', '+')
+      .replaceAll('_', '/');
+  }
+
+  String toUnpaddedBase64() {
+    return replaceAll('\n', '')
+      .replaceAll('=', '');
+  }
```

Comment:
How about?

```suggestion
  String toUnpaddedBase64() {
    return replaceAll(RegExp(r'=+$'), '');
  }
```

Reply:
I think the first one is safer, the one you suggested removing the '=' character at the end only.